packages/wonder-blocks-dropdown/src/components/option-item.tsx

```diff
@@ -220,31 +258,85 @@ export default class OptionItem extends React.Component<OptionProps> {
                         </LabelSmall>
                     ) : undefined
                 }
-                onClick={this.handleClick}
+                onClick={
+                    parentComponent !== "listbox" ? this.handleClick : undefined
+                }
                 {...sharedProps}
             />
         );
     }
+
+    render(): React.ReactNode {
+        const {disabled, focused, parentComponent, role, selected} = this.props;
+
+        if (parentComponent === "listbox") {
```

Comment:
**note:** I'm making these changes to add support to the new `Listbox` component, while supporting the existing use in `SingleSelect` and `MultiSelect` (used by `DropdownCore`). The long-term goal is to replace some parts of `DropdownCore` with `Listbox`. When that happens, I'll get rid of these customizations.